sympy/printing/precedence.py

```diff
@@ -47,6 +47,7 @@
     "KroneckerProduct": PRECEDENCE["Mul"],
     "Equality": PRECEDENCE["Mul"],
     "Unequality": PRECEDENCE["Mul"],
+    "Adjoint": PRECEDENCE["Pow"]
```

Comment:
... would be good to have someone else comment on this

Reply:
Actually, `Transpose` had also to be added here to make some combined expressions work.

Reply:
Hey, this is the wrong place. There is a `precedence_traditional` function for traditional math precedence. This is supposed to be the precedence of the `str` printer.

Reply:
The `str` printer prints `Adjoint` as a function:
```python
In [31]: Adjoint(A)
Out[31]:
 †
A 

In [32]: str(_)
Out[32]: 'Adjoint(A)'
```
I suppose `Adjoint` needs to have atomic precedence.